examples/structured_data/classification_with_tfdf.py

```diff
@@ -100,6 +87,8 @@
 
 # Target column name.
 TARGET_COLUMN_NAME = "income_level"
+# The labels of the target columns.
+TARGET_LABELS = [" - 50000.", " 50000+."]
 # Weight column name.
 WEIGHT_COLUMN_NAME = "instance_weight"
```

Comment:
Please remove.
See comment in https://github.com/keras-team/keras-io/pull/782

Reply:
The `TARGET_LABELS` is used later in the `prepare_dataframe` method.

Reply:
I am not using the attribute "as an input feature", rather as a weight to each instance. This will affect the loss computation; instances with higher weight will contribute more to the loss. This enforces the model to try to get the highly-weighted instances right, more than the low-weight instances, and I think it should be used as it represents the true distribution of the data.

Happy to discuss further

Reply:
I understood that.

However, do you think that `This attribute should *not* be used in the classifiers.` means that this field can still be used to do weighted training? This is effectively "using it" in the classifier.
I am not an expert in this dataset, so maybe is it that. Please confirm.